app/views/schools/add_participants/email.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :title, "Enter email to confirm" %>
+
+<% content_for :before_content, govuk_back_link(text: "Back", href: back_link_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl"><%= @school.name %> - <%= @cohort.display_name %> cohort</span>
+    <h1 class="govuk-heading-xl">What’s <%= add_participant_form.full_name %>’s email address?</h1>
+    <p class="govuk-body">We’ll use this to contact them for more information. This is needed to check their eligibility with the Teaching Regulation Authority.</p>
+    <%= form_for add_participant_form, url: { action: :update }, method: :patch do |f| %>
+      <%= f.govuk_text_field :email, label: { hidden: true }, width: "three-quarters", hint: { text: "You can enter their personal or school email address." } %>
```

Comment:
Are we sure we want `hidden: true` here? This results in the following markup:

```html
<label for="schools-add-participant-form-email-field" class="govuk-label">
    <span class="govuk-visually-hidden">Email</span>
</label>
```

meaning the label is visible to screenreaders

Reply:
@ethanmills just running this past Brian, (copying and pasting since he can't comment on Github) 'Labels should be visible to screen readers if there is accompanying text within the page - otherwise if its H1 and input then label can be used as heading. Is that what you mean?

Reply:
In this instance that's fine, because the label will read "Email". In other cases where the field has a multiword name, the label will be horrible :P ( [see this](https://govuk-form-builder.netlify.app/introduction/labels-captions-hints-and-legends/#:~:text=by%20default%2C%20the%20label%20text%20is%20set%20to%20the%20capitalised%20name%20of%20the%20attribute.%20if%20the%20attribute%20contains%20underscores%20they%20will%20be%20left%20in.%20this%20is%20intentional%20behaviour%20to%20prompt%20the%20developer%20to%20add%20some%20more%20suitable%20text.) )